app/components/Form/ApplicationForm.tsx

```diff
@@ -155,6 +156,9 @@ const ApplicationForm: React.FC<Props> = ({
     query
   );
   const { id, rowId, formData, status } = application;
+  
+  Settings.defaultZone = "America/Vancouver";
+  Settings.defaultLocale = "en-CA";
```

Comment:
should we do this in `pages/_app`, so that those settings are applied for every page?

Reply:
Done